rust/noosphere-core/src/data/address.rs

```diff
@@ -15,12 +15,15 @@ use super::{Did, IdentitiesIpld, Jwt, Link, MemoIpld};
 #[cfg(docs)]
 use crate::data::SphereIpld;
 
+/// The name of the fact (as defined for a [Ucan]) that contains the link for a
+/// [LinkRecord]
```

Comment:
nit: In the future, we should consider terminating punctuation on all doc comments

Reply:
Yah, I wonder if there is a spell check / style lint we can make use of for this kind of thing...